adapters/handlers/graphql/local/aggregate/hybrid_search.go

```diff
@@ -96,3 +140,73 @@ func hybridSubSearch(classObject *graphql.Object,
 		},
 	}
 }
+
+func nearTextFields(prefix string) graphql.InputObjectConfigFieldMap {
```

Comment:
This definition should be removed, we have unified module arguments in `github.com/weaviate/weaviate/usecases/modulecomponents` package, you can use this package to get the argument definition like this:
```
nearText.New(nearTextTransformer).Arguments()[nearText.NAME]
```
and this part should be added dynamically by `module provider` but this can be fixed later

Reply:
let's do this change in the next PR